File: exam-back/src/main/java/com/boblogservice/member/exception/NotCorrectTwoPasswordException.java
package com.boblogservice.member.exception;

import org.springframework.http.HttpStatus;
import org.springframework.web.bind.annotation.ResponseStatus;

@ResponseStatus(value = HttpStatus.NOT_FOUND, reason = "two password2 not correct")
public class NotCorrectTwoPasswordException extends RuntimeException {
    public NotCorrectTwoPasswordException(String message) {
        super(message);
    }
}

File: kotlin-back/src/main/kotlin/com/example/kotlinback/member/exception/NotCorrectTwoPasswordException.kt
package com.example.kotlinback.member.exception

import org.springframework.http.HttpStatus
import org.springframework.web.bind.annotation.ResponseStatus

@ResponseStatus(value = HttpStatus.NOT_FOUND, reason = "two password2 not correct")
class NotCorrectTwoPasswordException(message: String) : RuntimeException(message)
